SampleApp/package.json

```diff
@@ -22,7 +22,7 @@
     "react-native-moengage-geofence": "^4.0.0",
     "react-native-moengage-inbox": "^3.0.0",
     "react-native-render-html": "^6.3.4",
-    "react-native-safe-area-context": "3.3.2",
```

Comment:
is it required for iOS?

Reply:
@abhishek-engg-mobilesdk There was UI issue in iOS Sample app . so I have updated